plugins/outputs/graylog/graylog.go

```diff
@@ -457,9 +457,9 @@ func (g *Graylog) Write(metrics []telegraf.Metric) error {
 		}
 
 		for _, value := range values {
-			_, err := writer.Write([]byte(value))
+			_, err = writer.Write([]byte(value))
```

Comment:
Why this change? This increases the scope of `err`...

Reply:
> Why this change? This increases the scope of `err`...

It doesn't matter here. I'm not a fan of shadowed variables - using them too often can lead to tricky errors.